python/src/aiconfig/editor/client/src/Editor.tsx

```diff
@@ -68,6 +68,11 @@ export default function Editor() {
     });
   }, []);
 
+  const clearOutputs = useCallback(async () => {
+    return await ufetch.post(ROUTE_TABLE.CLEAR_OUTPUTS, {
```

Comment:
cc @rholinshead should we add a confirmation dialog that asks them if they're sure before deleting? I'm guessing no (especially if we have undo/redo actions in the future)

Reply:
I don't think we should since it's pretty explicit and not completely disruptive